f5router/f5router.go

```diff
@@ -575,6 +593,26 @@ func verifyDestAddress(va *bigipResources.VirtualAddress, partition string) (str
 	return "", fmt.Errorf("invalid address: %s", va.BindAddr)
 }
 
+// checkForString loops over a slice to see if a string exists in it
+func checkForString(list []string, text string) bool {
+	for _, val := range list {
+		if val == text {
+			return true
+		}
+	}
+	return false
+}
+
+func joinBigipPath(partition, objName string) string {
+	if objName == "" {
+		return ""
```

Comment:
would this be considered an error, or can it truly be an empty string?

Reply:
I think so, updated to reflect that.